### DISCUSSION TITLE
Linting/formatting/LSP support for JSX files by default

### DISCUSSION THREAD

**Discussion by daron81** [SpartanJ/ecode#65] [Q&A] [answered]
Hi there,

Are there any plans to add the above for JSX files by default? It was really easy to enable through the config, but it would be nice to have this ootb.

Thanks!

**Comment by SpartanJ** [comment, marked as answer]
Sure, that would be no problem. Can you share your configuration so I can use it? I don't use JSX, so I'm not sure if it should use the same configuration as JS/TS.
